app/models/frontend_router.rb

```diff
@@ -28,8 +28,10 @@ def query_info
       [ Variant, :id, ]
     when /evidence/, /evidence_items?/
       [ EvidenceItem, :id, ]
-    when /entrez/
```

Comment:
Even though its not totally clear, I think we should leave this `entrez` unfortunately as we don't want to break any links that currently work. We'd need to change the regex to be `entrez$` or similar so it doesn't match `entrez_name` accidentally.

The other option is that we could just keep `entrez` and then allow them to pass an ID or Symbol and do the resolution on our end.

Reply:
The API docs says that the "official" endpoint is `entrez_id` (https://griffithlab.github.io/civic-api-docs/#linking). With that in mind, I think we should actually make the behavior match the documentation and fix any broken links. If we want to support `entrez` instead of `entrez_id` then we should update the documentation to reflect that.

Reply:
Good catch, agreed! I didn't realize that's what the docs already said. With that in mind, I think this is good then.